contracts/NFTAuction.sol

```diff
@@ -655,8 +644,7 @@ contract NFTAuction {
         isFeePercentagesLessThanMaximum(_feePercentages)
     {
         if (_erc20Token != address(0)) {
-            nftContractAuctions[_nftContractAddress][_tokenId]
-                .ERC20Token = _erc20Token;
+            revert("Disabled ERC20 functionality");
```

Comment:
Here is the meat of it. 

Essentially whenever an auction is created, the _erc20Token needs to be left as address(0) to ensure its an ETH based auction. It will revert if this is not the case

Reply:
Yeah, just only allow ETH for all future auctions :+1:

Reply:
but why?